index.html

```diff
@@ -3688,39 +3679,10 @@ <h3>Evidence</h3>
         </p>
 
         <pre class="example nohighlight"
-          title="Usage of the evidence property">
```

Comment:
This example is being removed because the VCWG had agreed that any at risk features needed to have a concrete example of the usage of the feature. We use the Open Badges v3.0 example here.